src/server/processLauncher.ts

```diff
@@ -166,7 +166,7 @@ export function waitForLine(process: childProcess.ChildProcess, inputStream: str
         'Failed to launch browser!' + (error ? ' ' + error.message : ''),
         stderr,
         '',
-        'TROUBLESHOOTING: https://github.com/Microsoft/playwright/blob/master/docs/troubleshooting.md',
```

Comment:
Let's drop this line altogether as it makes little sense for troubleshooting anyway? We can refine the message later once we have meaningful instructions.